### PR TITLE
add new rpy2 version

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

**Comment by yhoogstrate** [comment]
LGTM, Thanks! 

Can I upload it to the testtoolshed for testing?

**Comment by bgruening** [comment]
Sure!
Oh what happened to this PR? So many old new stuff ... Please don't merge yet.

**Comment by bgruening** [comment]
Now it looks better!

**Comment by yhoogstrate** [comment]
@bgruening Not sure how I messed it up. Probably I some weird merges in my log? Anyway, I added a dependency to python because it installs via `python setup.py install` and I added a sha256sum to `<action type="download_by_url">`.

File: packages/package_rpy2_2_7_5/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <package name="R" version="3.2.1">
+        <repository name="package_r_3_2_1" owner="iuc" prior_installation_required="True" />
+    </package>
+    <package name="readline" version="6.2">
+        <repository name="package_readline_6_2" owner="iuc" prior_installation_required="True" />
+    </package>
+    <package name="rpy2" version="2.7.5">
+        <install version="1.0">
+            <actions>
+                <action type="download_by_url">https://pypi.python.org/packages/source/r/rpy2/rpy2-2.7.5.tar.gz</action>
```
Comment (erasche) [inline packages/package_rpy2_2_7_5/tool_dependencies.xml]: `0d88f7fff21fbef523c8678ed8f850a36052edca4ce702c62f234fdcb52de441`, if you wish to add it.

**Comment by yhoogstrate** [comment]
image
image
:+1:

**Pull request merged**